pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataSetup.java

```diff
@@ -143,8 +149,54 @@ public static void main(String[] args) throws Exception {
             // Ignore
         }
 
+        // Create public tenant
+        PropertyAdmin publicProperty = new PropertyAdmin();
+        byte[] publicPropertyDataJson = ObjectMapperFactory.getThreadLocal().writeValueAsBytes(publicProperty);
```

Comment:
I think we would still miss the `public/default` in standalone mode since it doesn't use this script. 

Standalone is creating `sample/standalone/ns1` by default when it starts up. We should just change that into `public/default`

Reply:
created a public/default namespace at standalone